website/layouts/index.ace

```diff
@@ -30,7 +30,7 @@ html lang={{.Site.LanguageCode}}
                 .col-sm-6
                   .panel.panel-primary
                     .panel-heading Proven at Twitter Scale
-                    .panel-body Heron powers all realtime analytics with varied use cases at Twitter for over 2 years. Incident reports dropped by an order of magnitude demonstrating proven reliability and scalability
+                    .panel-body Heron has powered all realtime analytics with varied use cases at Twitter for over 2 years. Incident reports dropped by an order of magnitude demonstrating proven reliability and scalability
```

Comment:
Good catch. If I could request one more edit please, we should replace "for over 2 years" with "since 2014".